SPECS/pyflakes/pyflakes.spec

```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Name:           pyflakes
 # WARNING: When updating pyflakes, check not to break flake8!
-Version:        2.5.0
+Version:        3.2.0
```

Comment:
seems odd that Source1 uses something with 2.0.0 version and our spec is up to 3.2.0.  do the other distros do this as well?

Reply:
Yes, this confused me a lot, but fedora does something similar with version 3.0.1 and uses 2.5.0 for Source1. I also tried the upgrade package pipeline and it does the same thing with the versions and sources.

Reply:
it looks like fedora is using a debian 2.5 for pyflakes 3.0.  why are we still on debian 2.0?

Reply:
it looks like there is a 3.1: https://deb.debian.org/debian/pool/main/p/pyflakes/

Reply:
Thanks, let me update that.

Reply:
It'd be nice to understand why it has two sources. :thinking:

Reply:
I corroborated that we use/need the debian source. Without it a test fails and we also use it for the information that the man command. I updated the source1 to use 3.0.